providers/os/resources/processes/testdata/aix72.toml

```diff
@@ -0,0 +1,41 @@
+[commands."ps -A -o pid,pcpu,pmem,vsz,tty,time,uid,args"]
+stdout = """     PID  %CPU  %MEM   VSZ     TT        TIME UID COMMAND
+     PID  %CPU  %MEM   VSZ     TT        TIME UID COMMAND
```

Comment:
nit: The header is twice here. Is this normal AIX output?

Reply:
copy & paste mistake. will update the test